ortho_seq_code/gui/job_runner.py

```diff
@@ -0,0 +1,42 @@
+from PyQt5.QtWidgets import QWidget
+import os
+
+from ortho_seq_code.orthogonal_polynomial import orthogonal_polynomial
+from ortho_seq_code.gui.worker import Worker
+
+
+class JobRunner(QWidget):
+    def __init__(self, parent, threadpool):
+        super(QWidget, self).__init__(parent)
+        self.parent = parent
+        self.threadpool = threadpool
+
+    def job_func(self, progress_callback):
+        orthogonal_polynomial(
+            self.filename,
+            self.pheno_file,
+            self.molecule,
+            2,
+            4,
+            1,
```

Comment:
I think we can get rid of these now since the cli automatically calculates the values for these flags.

Reply:
nvm, since the gui_draft branch doesn't have the new changes, we can keep this for now.